workflow/rocoto/gfs_tasks.py

```diff
@@ -672,7 +672,11 @@ def marinebmat(self):
         data = f'{ocean_hist_path}/gdas.ocean.t@Hz.inst.f009.nc'
         dep_dict = {'type': 'data', 'data': data, 'offset': f"-{timedelta_to_HMS(self._base['interval_gdas'])}"}
         deps.append(rocoto.add_dependency(dep_dict))
-        dependencies = rocoto.create_dependency(dep=deps)
+        if self.options['do_hybvar']:
```

Comment:
Should this be `do_hybvar_ocn`?  This option probably needs to be added to [applications/gfs_cycled.py:GFSCycledAppConfig._get_run_options](https://github.com/NOAA-EMC/global-workflow/blob/6b2d84ed5fd05fa6df26456b7e0bd6e1b680811f/workflow/applications/gfs_cycled.py#L32).

Reply:
yes, good catch.

Reply:
```suggestion
        if self.options['do_hybvar_ocn']:
```

Reply:
that's weird, I just pushed that exact same fix from my local sandbox ... humm ...